samples/automl/automlVisionDataset.js

```diff
@@ -0,0 +1,400 @@
+  // const datasetId = `Id of the dataset, e.g. "ICN3874392874098"`;
+  // const path = `Cloud Storage Uri, target must be in AutoML Vision CSV format, e.g. “gs://myproject/mytraindata.csv”;`
+
+  // Get the full path of the dataset.
+  const datasetFullId = client.datasetPath(projectId, computeRegion, datasetId);
+
+  // Get the multiple Google Cloud Storage URIs.
+  const inputUris = path.split(`,`);
+  const inputConfig = {
+    gcsSource: {
+      inputUris: inputUris,
+    },
+  };
+
+  // Import the dataset from the input URI.
+  client
+    .importData({name: datasetFullId, inputConfig: inputConfig})
+    .then(responses => {
+      const operation = responses[0];
+      console.log(`Processing import...`);
+      return operation.promise();
+    })
+    .then(responses => {
+      // The final result of the operation.
+      if (responses[2].done === true) console.log(`Data imported.`);
+    })
+    .catch(err => {
+      console.error(err);
+    });
+  // [END automl_vision_importData]
+}
+
+function exportData(projectId, computeRegion, datasetId, outputUri) {
+  // [START automl_vision_exportData]
+  const automl = require(`@google-cloud/automl`);
+
+  const client = new automl.v1beta1.AutoMlClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const projectId = `numeric Id of the project from PROJECT_ID env variable, e.g. 123456`;
+  // const computeRegion = `region-name, e.g. "us-central1"`;
+  // const datasetId = `Id of the dataset, e.g. "ICN3874392874098"`;
+  // const outputUri = `Google Cloud Storage URI for the export directory, e.g. “gs://myproject/output”;`
+
+  // Get the full path of the dataset.
+  const datasetFullId = client.datasetPath(projectId, computeRegion, datasetId);
+
+  // Set the output URI
+  const outputConfig = {
+    gcsDestination: {
+      outputUriPrefix: outputUri,
+    },
+  };
+
+  // Export the data to the output URI.
+  client
+    .exportData({name: datasetFullId, outputConfig: outputConfig})
+    .then(responses => {
+      const operation = responses[0];
+      console.log(`Processing export...`);
+      return operation.promise();
+    })
+    .then(responses => {
+      // The final result of the operation.
+      if (responses[2].done === true) console.log(`Data exported.`);
+    })
+    .catch(err => {
+      console.error(err);
+    });
+  // [END automl_vision_exportData]
+}
+
+function deleteDataset(projectId, computeRegion, datasetId) {
+  // [START automl_vision_deleteDataset]
+  const automl = require(`@google-cloud/automl`);
+
+  const client = new automl.v1beta1.AutoMlClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const projectId = `numeric Id of the project from PROJECT_ID env variable, e.g. 123456`;
+  // const computeRegion = `region-name, e.g. "us-central1"`;
+  // const datasetId = `Id of the dataset, e.g. "ICN3874392874098"`;

```

Comment:
How about `'Id of the dataset, e.g. "ICN0123456789"'` or just `'Id of the dataset'`